### PR TITLE
docker: initial release

### PR DESCRIPTION
**Pull request by tiborsimko**
* b432ac3384a46985eaab0f82b7fa5544fdaebd87 setup: PEP-8 and PEP-440 improvements
* eb250fe019c26524fbbfb87a8042b373565de273 docker: initial release

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with intbitset; if not, write to the Free Software Foundation,
+# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this licence, CERN does not waive the privileges and
+# immunities granted to it by virtue of its status as an
+# Intergovernmental Organization or submit itself to any jurisdiction.
+
+web:
```
Comment (jirikuncar) [inline docker-compose.yml]: should we call it `worker`, `intbitset` or just `node`?

File: setup.py [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+# You should have received a copy of the GNU General Public License
+# along with intbitset; if not, write to the Free Software Foundation,
+# Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
 
-from setuptools import setup
+"""intbitset: C-based extension implementing fast integer bit sets."""
```
Comment (jirikuncar) [inline setup.py]: what about reusing it in short description via `__doc__`

--
I would also remove the prefix `intbitset: `

File: docker-compose.yml (hunk 1 above)
Comment (tiborsimko) [inline docker-compose.yml]: Could be also `app`, perhaps. I kept the usual `web` by momentum.  Not sure it matters in this case, actually. Besides the usual web/db/worker/redis, are container names standardised?

File: setup.py (hunk 2 above)
Comment (tiborsimko) [inline setup.py]: Good idea; repushed.

File: docker-compose.yml (hunk 1 above)
Comment (tiborsimko) [inline docker-compose.yml]: (BTW for "jsonalchemy" we kept "web" as well.)

**Pull request merged**